website/src/components/newsletter-glow-container/newsletter-glow-container.tsx

```
@@ -0,0 +1,32 @@
+'use client';
+
+import { LanguageCode } from '@socialincome/shared/src/types/language';
+import { GlowHoverContainer, Typography } from '@socialincome/ui';
+
+import NewsletterForm from '@/components/newsletter-form/newsletter-form';
+import { NewsletterPopupTranslations } from '@/components/newsletter-popup/newsletter-popup-client';
+
+type NewsletterForm = {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'NewsletterForm'. Consider to delete it or rename it.

'NewsletterForm' is defined here:


(lint/suspicious/noRedeclare)